<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgDistances</class>
 <widget class="QDialog" name="CQDlgDistances">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distances</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="qqEnableAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Enable distance calculations</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QListWidget" name="qqDistanceList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Distance object properties</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QToolButton" name="qqAdjustSegmentColor">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Adjust segment color</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqUseThreshold">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Use threshold [m]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Segment width [pt]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqThreshold">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqSegmentWidth">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>40</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqExplicitHandling">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Explicit handling</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqDisplaySegment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Display dist. segment</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="qqAddNewObject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Add new distance object</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqAddNewObject</tabstop>
  <tabstop>qqDistanceList</tabstop>
  <tabstop>qqExplicitHandling</tabstop>
  <tabstop>qqUseThreshold</tabstop>
  <tabstop>qqThreshold</tabstop>
  <tabstop>qqDisplaySegment</tabstop>
  <tabstop>qqSegmentWidth</tabstop>
  <tabstop>qqAdjustSegmentColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
